"Attribute could not be mapped to GraphQL" with @Enumerated Enum

As seen in: https://github.com/timtebeek/graphql-jpa-enum
When using the following:

```
public enum Genre {
    DRAMA, HORROR, MYSTERY, THRILLER
}
```

```
@Entity
public class Book {
    @Id
    Long id;

    String title;

    @ManyToOne
    Author author;

    @Enumerated(EnumType.STRING)
    Genre genre;
}
```

And loading my application context in a test, I get the following stacktrace:

```
...
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.crygier.graphql.GraphQLExecutor]: Factory method 'graphQLExecutor' threw exception; nested exception is java.lang.UnsupportedOperationException: Attribute could not be mapped to GraphQL: org.hibernate.jpa.internal.metamodel.SingularAttributeImpl@54ae1240
    at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:189) ~[spring-beans-4.3.3.RELEASE.jar:4.3.3.RELEASE]
    at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:588) ~[spring-beans-4.3.3.RELEASE.jar:4.3.3.RELEASE]
    ... 42 common frames omitted
Caused by: java.lang.UnsupportedOperationException: Attribute could not be mapped to GraphQL: org.hibernate.jpa.internal.metamodel.SingularAttributeImpl@54ae1240
    at org.crygier.graphql.GraphQLSchemaBuilder.getAttributeType(GraphQLSchemaBuilder.java:151) ~[classes/:na]
    at org.crygier.graphql.GraphQLSchemaBuilder.getObjectField(GraphQLSchemaBuilder.java:96) ~[classes/:na]
...
```

Trace debugging reveals that the expection is raised for `Book.genre`. How can the model generation be updated to allow for such a mapped enum field?

--- a/src/main/java/org/crygier/graphql/GraphQLSchemaBuilder.java
+++ b/src/main/java/org/crygier/graphql/GraphQLSchemaBuilder.java
@@ -137,6 +137,9 @@ public class GraphQLSchemaBuilder {
                 return Scalars.GraphQLLong;
             else if (Boolean.class.isAssignableFrom(attribute.getJavaType()) || boolean.class.isAssignableFrom(attribute.getJavaType()))
                 return Scalars.GraphQLBoolean;
+            else if (attribute.getJavaType().isEnum()) {
+                return getTypeFromJavaType(attribute.getJavaType());
+            }
         } else if (attribute.getPersistentAttributeType() == Attribute.PersistentAttributeType.ONE_TO_MANY || attribute.getPersistentAttributeType() == Attribute.PersistentAttributeType.MANY_TO_MANY) {
             EntityType foreignType = (EntityType) ((PluralAttribute) attribute).getElementType();
             return new GraphQLList(new GraphQLTypeReference(foreignType.getName()));

--- a/src/main/java/org/crygier/graphql/GraphQLSchemaBuilder.java
+++ b/src/main/java/org/crygier/graphql/GraphQLSchemaBuilder.java
@@ -131,7 +131,8 @@ public class GraphQLSchemaBuilder {
                 return Scalars.GraphQLString;
             else if (Integer.class.isAssignableFrom(attribute.getJavaType()) || int.class.isAssignableFrom(attribute.getJavaType()))
                 return Scalars.GraphQLInt;
-            else if (Float.class.isAssignableFrom(attribute.getJavaType()) || float.class.isAssignableFrom(attribute.getJavaType()))
+            else if (Float.class.isAssignableFrom(attribute.getJavaType()) || float.class.isAssignableFrom(attribute.getJavaType())
+                    || Double.class.isAssignableFrom(attribute.getJavaType()) || double.class.isAssignableFrom(attribute.getJavaType()))
                 return Scalars.GraphQLFloat;
             else if (Long.class.isAssignableFrom(attribute.getJavaType()) || long.class.isAssignableFrom(attribute.getJavaType()))
                 return Scalars.GraphQLLong;

